geonames/track.json

```diff
@@ -359,6 +359,29 @@
       ]
     },
     {
+      "name": "append-fast-sorted no-conflicts",
```

Comment:
I think this challenge should be named "append-fast-sorted-no-conflicts" (i.e. without any spaces). This makes it easier to type the name on the command line (no quoting required).